src/api.js

```diff
@@ -0,0 +1,13 @@
+export const API_ROOT = process.env.REACT_APP_API_ROOT
+export const API_KEY = process.env.REACT_APP_API_KEY
+
+export const params = {
+  'api-key': `${API_KEY}`
+}
+
+// NOTE: allows for additional parameters
+export const get = async(url) => {
+  url += '?'
+  Object.entries(params).map(([key, val]) => (url += `&${key}=${val}`))
```

Comment:
Consider using a more generic iteration function, like `forEach`, since nothing is being mapped to.

Reply:
I used the `map` function since it actually executes (very slightly) faster than the `forEach` method. Actually, a `for ( )` loop would be even faster 🤔